src/overlays/actors/ovl_Bg_Ikana_Ray/z_bg_ikana_ray.c

```diff
@@ -69,7 +69,7 @@ void BgIkanaRay_Init(Actor* thisx, GlobalContext* globalCtx) {
     Collider_SetCylinder(globalCtx, collision, &this->actor, &sCylinderInit);
     Collider_UpdateCylinder(&this->actor, &this->collision);
 
-    this->animatedTextures = (AnimatedMaterial*)Lib_SegmentedToVirtual(object_ikana_obj_Matanimheader_001228);
+    this->animatedTextures = Lib_SegmentedToVirtual(object_ikana_obj_Matanimheader_001228);
 
     if (Flags_GetSwitch(globalCtx, this->actor.params & 0x7F)) {
```

Comment:
Since this file is already being changed, would it be convenient to make a macro for this? I understand these clean up PRs can have a lot of scope creep.

Reply:
Thanks for pointing it out. Added the macro.